ui/src/app/core/settings.service.ts

```diff
@@ -73,7 +90,8 @@ export class SettingsService {
   private createEmptySettingsForProject(projectId: string): ProjectSettings {
     this.currentSettings.v1.projects.push({
       'projectId': projectId,
-      'displayColumns': null
+      'displayColumns': null,
```

Comment:
I think omitting these keys would be a better simulation of "not having settings". This is what older versions of the settings will look like.

Reply:
When I take `displayColumns` and `pageSize` out of this function, I get the error
> Argument of type '{ projectId: string; }' is not assignable to parameter of type 'ProjectSettings'.
>  Property 'displayColumns' is missing in type '{ projectId: string; }'.